staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -301,17 +301,20 @@ func (le *LeaderElector) tryAcquireOrRenew() bool {
 
 	// 1. obtain or create the ElectionRecord
 	oldLeaderElectionRecord, err := le.config.Lock.Get()
+
 	if err != nil {
 		if !errors.IsNotFound(err) {
 			klog.Errorf("error retrieving resource lock %v: %v", le.config.Lock.Describe(), err)
 			return false
 		}
+		// Try to create the lock
 		if err = le.config.Lock.Create(leaderElectionRecord); err != nil {
 			klog.Errorf("error initially creating leader election record: %v", err)
 			return false
 		}
 		le.observedRecord = leaderElectionRecord
 		le.observedTime = le.clock.Now()
+		klog.Info("Created a new lock, and acquired it successfully at time: %v", le.observedTime)
```

Comment:
"Created a new leader election record" ?  "Lock" is a bit vague.

"successfully acquired lease" or "successfully renewed lease" messages are already logged after this, are they sufficient or can they be improved to log what is needed?

Reply:
fixed